.env

```diff
@@ -5,3 +5,5 @@ PROMETHEUS_PORT=9090
 CONSOLE_VER=0.1.1
 PROMETHEUS_VER=v2.4.3
 GRAFANA_VER=5.3.1
+
+MONGOOSE_IP=http://localhost:9999
```

Comment:
this doesn't look like ip address

Reply:
Resolved at https://github.com/emc-mongoose/console/pull/41/commits/8b223431cc5fd5e5c37be10a06078ffa4be6e96c